### ISSUE TITLE
Failing on both second jobs

### ISSUE THREAD

**Issue by zrachlin** [anna-geller/dataflow-ops#5]
I'm attempting to follow [this](https://towardsdatascience.com/prefect-aws-ecs-fargate-github-actions-make-serverless-dataflows-as-easy-as-py-f6025335effc#8d94) recipe and running into errors with both of the concurrent second jobs in the "1 Initial - ECS Prefect agent" action. I set the AWS and Prefect secrets as described in the article, and I can see the container in AWS ECR got created successfully, but not sure about both of the errors shown in the images below:

<img width="1477" alt="image" src="https://user-images.githubusercontent.com/29631035/204789244-5b6d7d8e-05a3-4e93-ac24-afb9c2ad9cf1.png">
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/29631035/204794631-1a90416e-eb49-4c86-904a-10ff3067d1dd.png">

I'm also not seeing any of the blocks show up in the Prefect UI, even though they say they registered and should be visible:
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/29631035/204795202-14bc3c4b-b1ed-472e-85e0-3421637d6d83.png">
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/29631035/204795370-fb606907-415a-48fd-ae82-e4fedf42e82d.png">
I know that the action fails before getting to the "Create Blocks and Upload to S3" step, but the message shown above in the "Python Dependencies" step led me to believe that I should see them in the Prefect UI.

I've run the delete action to try to wipe things out and start over, but no luck so far. Any help would be greatly appreciated!

**Comment by anna-geller** [comment]
interesting - looks like the login to Cloud didn't work because Secret wasn't passed properly - did you add PREFECT_API_KEY key and PREFECT_API_URL and do they look similar to this format?

```
PREFECT_API_URL = "https://api.prefect.cloud/api/accounts/xxxx/workspaces/xxxx"
PREFECT_API_KEY = "pnu_xxxx"
```

just making sure, did you create an S3 bucket and pass its name as input?

**Comment by anna-geller** [comment]
I ran it just now and this step seems to work
![image](https://user-images.githubusercontent.com/86264395/204811303-93ecb32e-977b-488c-82f4-1bb731a598d0.png)


it looks like either API key expired or has not been properly added to secrets somehow, can you cross-check and try again? maybe create a new fresh API key just to be sure?

can you confirm that login into cloud from your local CLI works using that API key and workspace?

**Comment by zrachlin** [comment]
I added quotes around the PREFECT_API_URL and PREFECT_API_KEY values and that got me through the "Prefect Cloud login" step, but now I'm getting this:
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/29631035/204840781-5ff3938f-ecbf-4992-a8cd-21af8bb69351.png">
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/29631035/204840890-93d8bb33-25e7-469c-914e-106262f3cd53.png">

**Comment by anna-geller** [comment]
you shouldn't need quotes around the PREFECT_API_URL and PREFECT_API_KEY values - can you try the same values from your local terminal on your laptop?

```
pip install prefect
prefect cloud login
```
From here in your terminal, you can finish authentication and API key creation from a browser. 
Run an example flow and ensure you can see its execution in your Prefect Cloud UI to validate the setup is working.

Then, print the values that you got in your CLI profile:
```
cat ~/.prefect/profiles.toml
```
If that was successful, use the same API key and URL values in GH Actions. 

LMK if that helped sort that out.

**Comment by zrachlin** [comment]
That worked, thanks! I think I figured out what was causing the issue -> I originally ran `prefect config view` from my terminal to get the PREFECT_API_URL, and I copied and pasted that value directly into the github secrets - apparently there was a linefeed (\n) character in the middle of it that got url-encoded to %0a when github saved it. Copying and pasting from `cat ~/.prefect/profiles.toml` didn't have that issue.

Just FYI, when I tried authenticating through the browser, I got this:
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/29631035/204849826-e5d7a000-a7e9-4cc7-a89c-a9bb668dee77.png">
I was able to authenticate successfully through the CLI by pasting my API key into the terminal.
